lib/metadata.py

```diff
@@ -168,7 +168,7 @@ def print_dates(dates_list):
                             dates = "?"
                         print(f"- {location} ({n} pic{picx(n)}) [{dates}]")
 
-        softwares = self.softwares
+        softwares = self.software
         if softwares:
             print(f"\n[+] {len(softwares)} software{picx(len(softwares))} found !")
             for software, data in softwares.items():
```

Comment:
Ok for the changes, but I don't want to change the variable in this file, this is more understandable like this, so we know there are several softwares.